.github/workflows/ci2.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           fail_on: nothing
 
   Build:
-    name: Build DAOS
+    name: Build DAOS (x86_64)
```

Comment:
This is a change to the job name which is what GitHub reports it as, and is marked as required/not required.  To land this PR with this change in place we'd need to untick the checks as required, wait a week, land this PR, then wait another week and finally mark the new checks as required.